src/engine/interfaces.ts

```diff
@@ -0,0 +1,40 @@
+export enum RAFPhase {
+  MEASURE,
+  MUTATE
+}
+
+export interface QueueElementInterface {
```

Comment:
I think I will leave this just to be more explicit, but I did not realize typescript actually infers that sort of thing, good to know.